.rest

```diff
@@ -0,0 +1,31 @@
+POST http://localhost:3000/user_signup
```

Comment:
don't commit this file - it's for your testing alone

Reply:
I've added it to .gitignore Idk why it ended up here